tests/plugins/balloontoolbar/context/focusablebuttons.js

```diff
@@ -0,0 +1,87 @@
+/* bender-tags: balloontoolbar */
+/* bender-ckeditor-plugins: balloontoolbar, basicstyles, button, toolbar */
+
+( function() {
+	'use strict';
+
+	bender.editor = {
+		startupData: '<p>Hello <strong>world</strong>.</p>',
+		config: {
+			extraAllowedContent: 'strong em'
+		}
+	};
+
+	bender.test( {
+		'test check if balloon toolbar elements are registered as focusable': function() {
+			var editor = this.editor,
+				toolbar = new CKEDITOR.ui.balloonToolbar( editor ),
+				before,
+				afterAdding,
+				afterDeleting;
+
+			toolbar.addItem( 'Bold', new CKEDITOR.ui.button( {
+				label: 'Bold',
+				command: 'bold'
+			} ) );
+
+			before = CKEDITOR.tools.objectKeys( toolbar._view.focusables );
+
+			toolbar.attach( editor.editable().findOne( 'strong' ) );
+			afterAdding = CKEDITOR.tools.objectKeys( toolbar._view.focusables );
+
+			toolbar.deleteItem( 'Bold' );
+			afterDeleting = CKEDITOR.tools.objectKeys( toolbar._view.focusables );
+
+			assert.isFalse( CKEDITOR.tools.arrayCompare( before, afterAdding ), 'After adding new button, there should appear new focusables.' );
```

Comment:
Assertion messages should not end with a dot character. Please fix also other occurences.